ONS-International-trade-in-Services

santhosh-thangavel opened the pull request:
1.Added cubes class
2.Removed warnings

Shannon95 commented on datasets/ONS-International-Trade-in-Services/main.py:
```diff
@@ -149,7 +150,6 @@ def process_tab(tab):
                                                               }.get(x, x))
 
 observations['Value'] = observations['Value'].round(decimals=2)
-# -
```
Start review

Shannon95 reviewed (commented):
I believe some of the changes have been merged into the master already before a proper review, apologies my fault. Few aspects to change/make a note on the overall notebook which I will comment here and try to reference exactly where it relates to. I understand some these mentioned are legacy code. 

- Remove line 23 `scraper.dataset.family = info["families"]` 
scraper = Scraper(seed="info.json") will pick up the url and family information from the info.json automatically.

- Change line 27 to `tabs = scraper.distribution(latest=True).as_databaker()`
specifically selecting the latest distribution instead of selecting by index. 

move and adapt line 141 `cubes.add_cube(scraper, observations, "International trade in services")` to after the post-processing has taken place e.g. after the Measure Type and Unit column has been dropped to : 
`cubes.add_cube(scraper, observations.drop_duplicates(), "International trade in services")` 

- Add Tracer into transformation, doesn't seem to be currently implemented.

santhosh-thangavel commented:
1. Reverted back to old code
2. Made changes to pick up url & family information from info.json
3. Latest distribution is selected specifically instead of selection by index
4. Moved add_cube function at the right place

Shannon95 commented:
Happy for you to merge this in @santhosh-thangavel
